bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/internal/ThingManager.java

```diff
@@ -162,8 +162,12 @@ public void statusUpdated(final Thing thing, ThingStatusInfo thingStatus) {
                 Bridge bridge = (Bridge) thing;
                 for (Thing bridgeThing : bridge.getThings()) {
                     if (thingStatus.getStatus() == ThingStatus.ONLINE) {
-                        ThingStatusInfo statusInfo = ThingStatusInfoBuilder.create(ThingStatus.ONLINE).build();
-                        setThingStatus(bridgeThing, statusInfo);
+                    	ThingStatusInfo bridgeThingStatus = bridgeThing.getStatusInfo(); 
+                    	if (bridgeThingStatus.getStatus() == ThingStatus.ONLINE || 
```

Comment:
not according to me, but when I had in in a recent PR for the MAX binding a check for the to not do another update he commented:
> This check should not be done. It should be set to ONLINE, even if it already is online - but we can be sure we have a more recent timestamp for this updat

Hence I assumed he wanted these kind of updates to the same status.
Personally I feel these updates are not too useful and the info logging of it are cluttering the logs. #430